spec/features/cart/remove_from_quantity_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe "When a user looks at their cart" do
```

Comment:
Test looks good. I found it was helpful to put all the item/merchant creation and adding items inside of a `before each`